Gordon360/Services/ProfileService.cs

```diff
@@ -73,14 +73,14 @@ public MailboxViewModel GetMailboxCombination(string username)
     /// get a user's birthday
     /// </summary>
     /// <param name="username">The username of the person to get the birthdate of</param>
-    /// <returns>Date the user's date of birth</returns>
+    /// <returns>Date the user's date of birth, if available, or a default time of 0 ticks.</returns>
     public DateTime GetBirthdate(string username)
     {
         var birthdate = context.ACCOUNT.FirstOrDefault(a => a.AD_Username == username)?.Birth_Date;
 
         if (birthdate == null)
         {
-            throw new ResourceNotFoundException() { ExceptionMessage = "A birthday was not found for this user." };
+            return new DateTime(0L);  // Ignore missing birthday.
```

Comment:
```suggestion
            return new DateTime();  // Ignore missing birthday.
```
I'm pretty sure default datetime is 1/1/0001 00:00:00

Reply:
yes, and done.